.github/workflows/tests.yml

```diff
@@ -19,7 +19,7 @@ on:
         'Dockerfile',
       ]
   pull_request:
-    branches: [main]
+    branches: [main, transaction-and-log]
```

Comment:
forgot to rm this?

Reply:
Added it to force the test suite to run since it is not based on `main` but `transaction-and-log` branch